### DISCUSSION TITLE
❓ [Question] Has anyone exported detectron2?

### DISCUSSION THREAD

**Discussion by cbasavaraj** [pytorch/TensorRT#241] [Q&A]
## ❓ Question

Hi, has anyone successfully exported detectron2 using TRTorch? Ideally the whole network from input list of images to output Instances. But I guess even just the backbone and other time consuming operations like RoIAlign can be useful.

I am yet to try myself, but found nothing in the Issues, so just checking. Thanks

**Comment by batrlatom** [comment]
This would be great since detectron team is very resistent to tensorrt I think :+)
